### PR TITLE
Modify input files in tests

### PR DESCRIPTION
**Pull request by pnbabu**
Modify integration tests to run from any directory by providing absolute paths of input files to `generate_nest_target()` function.

### CODE REVIEW

File: tests/nest_tests/stdp_triplet_synapse_test.py [hunk 1]
```diff
@@ -18,6 +18,7 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+import os
```
Comment (clinssen) [inline tests/nest_tests/stdp_triplet_synapse_test.py:21]: we should move towards import statements ordered according to:

[import python standard library]

[import our own things, like nest and pynestml]

so in this case probably something like

```python
import numpy as np
import os
import pytest
```

**Review by clinssen — COMMENTED** [review]
Thanks for the PR! Just a few minor cleanups for consistency.

File: tests/nest_tests/stdp_triplet_synapse_test.py (hunk 1 above)
Comment (pnbabu) [reply tests/nest_tests/stdp_triplet_synapse_test.py]: Do you also mean to order them alphabetically?

**Comment by clinssen** [comment]
Great, many thanks!

**Pull request merged**